chsdi/models/vector/stopo.py

```diff
@@ -494,6 +494,7 @@ class SwissboundariesGemeinde(Base, Vector):
     gemflaeche = Column('gemflaeche', Numeric)
     perimeter = Column('perimeter', Numeric)
     kanton = Column('kanton', Unicode)
+    objektart = Column('objektart', Numeric)
```

Comment:
Please take the chance to replace Numeric, prefer float or integer. Only use it if really needed